models/enterprise_sonic/interfaces/overridden_example_01.txt

```diff
@@ -5,25 +5,36 @@
 #
  show interface status | no-more
 ------------------------------------------------------------------------------------------
-Name                Description         Admin          Oper           Speed          MTU
+Name                Description         Admin     Oper      AutoNeg     Speed        MTU
 ------------------------------------------------------------------------------------------
-Ethernet0           E0                  up                            100000         9100
-Ethernet4           E4                  up                            100000         9100
-Ethernet8           E8                  down                          100000         9100
-Ethernet12          -                   down                          1000           9100
-Ethernet16          -                   down                          100000         9100
+Ethernet0           E0                  up                              100000       9100
+Ethernet4           E4                  up                              100000       9100
+Ethernet8           E8                  down                            100000       9100
+Ethernet12          -                   down                            1000         9100
+Ethernet16          -                   down                            100000       9100
+
+ show running-configuration interface Ethernet 8
```

Comment:
In this file, also, the "before" and "after" output of the "show running-configuration interface" command for all three interfaces (Ethernet8, Ethernet12, and Ethernet 16) would help to show how this "state" operates.